jobs/kubelet/templates/bin/drain.erb

```diff
@@ -70,6 +63,7 @@ check_if_pidfile_exists() {
 }
 
 drain_kubelet() {
+  local node_name=$(kubectl --kubeconfig /var/vcap/jobs/kubeconfig/config/kubeconfig get nodes -o wide | grep "<%= spec.ip %>" | awk '{print $1}')
```

Comment:
No. It will return the node name. The `-o wide` allows us to grep on the IP but the $1 will be the first element returned which is the node name.